modules/@apostrophecms/asset/lib/webpack/apos/webpack.config.js

```diff
@@ -22,6 +23,7 @@ module.exports = ({
     )
   );
 
+  const mode = process.env.NODE_ENV || 'development';
```

Comment:
wouldn't it be safer to keep it in the function?

Reply:
It is still in the function.. Just moved it out of the config const to pass it to the `getVueAlias` function.

Reply:
ok, my bad!